### DISCUSSION TITLE
Discussion avec Lucas & Mathis - 10/10/2023

### DISCUSSION THREAD

**Discussion by AntoinePiron** [org-SCAN/react-app#4] [Meeting Report]
### Gestion des teams et de l'auth ? 🔒

Définir un "super admin" qui lui même pourra setup des admins régionaux &rarr; permet de segmenter la gestion des comptes par région.
On pourrait ensuite tourner sur un auth made in SCAN.

### Format de donnée lors d'un appel API de forms 🌐

Eux fonctionnait avec le composant android directement. Mais possible de récuperer la data_type value car elle arrive avec aussi. Donc pas forcément nécessaire de revoir l'appel.

### Discussion sur la langue 🗣️

Permettre le multilingue. Se baser sur la langue système mais permettre la modification dans les paramètres.

**Comment by skeayz** [comment]
J'ai à peu près les même notes
